### DISCUSSION TITLE
'Request a number' option on search-and-buy page.

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#2109] [Office hours] [answered]
I'm working on an option to inform users about the 'request a number form' when they are searching for a number type or capability that is disabled.
The idea is for this link to be hidden but available if customers want to submit the request as a last ressource because in the past, when it was visible, customers just went straight to this option without searching for the numbers. 
Here are the [4 options](https://www.figma.com/proto/hLWQ2TeMdrIrZsnozKNzOa/Buying-a-number-flow?page-id=1401%3A44003&node-id=1401%3A52539&viewport=380%2C48%2C0.19&scaling=min-zoom&starting-point-node-id=1401%3A52539&show-proto-sidebar=1) I'm exploring. 
And as a fifth I was thinking of a way to either place a button on a tooltip for a disabled checkbox, or make the disabled checkbox clickable so that the popover appears... but I'm not sure if this a good idea thinking about accesibility.

**Comment by shleewhite** [comment, marked as answer]
@lgalloherran came to design system office hours and we discussed putting the "Can't find a number" link near the country dropdown so the information isn't tied to just one field.

We also talked about moving the text that shows up when you hover a disabled checkbox to help text below the checkbox group.
